subsys/mgmt/mcumgr/lib/cmd/img_mgmt/include/img_mgmt/image_info.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2018-2021 mcumgr authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef H_IMG_MGMT_INFO_
+#define H_IMG_MGMT_INFO_
+
+#include <inttypes.h>
+#include "image.h"
+#include "mgmt/mgmt.h"
+#include <zcbor_common.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* 255.255.65535.4294967295\0 */
+#define IMG_MGMT_VER_MAX_STR_LEN	(sizeof("255.255.65535.4294967295"))
+
+/*
+ * @brief Read info of an image give the slot number
+ *
+ * @param image_slot	 Image slot to read info from
+ * @param image_version  Image version to be filled up
+ * @param hash		   Ptr to the read image hash
```

Comment:
Odd alignment

Reply:
I am just moving the stuff, which makes git comparison hard to figure out what has also been modified.
I can add additional commit that will be fixing the style, but I want to avoid doing the move and style fix in single commit as it looks like things get sneaked into moved code/comments.